appinfo/info.xml

```diff
@@ -2,10 +2,11 @@
 <info>
     <id>contacts</id>
     <name>Contacts</name>
-    <summary>A contacts app for ownCloud</summary>
+    <summary>Manage and sync your contacts across multiple devices</summary>
```

Comment:
the contacts app has nothing to do with sync and mutliple devices - this is a backend functionality provided by core.